manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -25,7 +25,8 @@ spec:
         - alert: PodDisruptionBudgetAtLimit
           annotations:
             summary: The pod disruption budget is preventing further disruption to pods.
-            description: The pod disruption budget is at the minimum disruptions allowed level. The number of current healthy pods is equal to the desired healthy pods.
+            description: |-
+              The {{ $labels.poddisruptionbudget }} pod disruption budget in the {{ $labels.namespace }} namespace is at the maximum allowed disruption. The number of current healthy pods is equal to the desired healthy pods.{{ with $console_url := "console_url" | query }}{{ if ne (len (label "url" (first $console_url))) 0}} For more information refer to {{ label "url" (first $console_url) }}/k8s/ns/{{ $labels.namespace }}/poddisruptionbudgets/{{ $labels.poddisruptionbudget }}{{ end }}{{ end }}
```

Comment:
Then we would not need the console url and let the console handle all the link rendering. It would also make the description nicer when looking at the alert detail in `/monitoring/alertrules/1234`.